contrib/kind-dual-stack-conversion.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
remove "FOR DEVELOPMENT ONLY NOT FOR TESTING" since we are using this in our presubmit job right? It's fine that its bash, as long as it gets the job done. Just add set -eux so if a command fails the test fails.